### PR TITLE
Maggie/get started

### PR DESCRIPTION
**Pull request by YiyueMaggieMao**

### CODE REVIEW

File: client/src/pages/Connect/Connect.css [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+}
+
+#get-started-form {
+    margin-right: 5vw;
+}
+
+#get-started-form-mobile {
+    width: 80vw;
+}
+
```
Comment (nathansit) [inline client/src/pages/Connect/Connect.css:26]: I'm not sure where exactly, but @chauandrew had a way of making the horizontal scroll bar not just disappear, but no longer necessary?

File: client/src/pages/Connect/Connect.css [hunk 2]
```diff
@@ -0,0 +1,35 @@
+.get-started-banner {
+    width: 100vw;
+    height: 36vw;
+}
+
+#get-started-body {
+    display: flex;
```
Comment (nathansit) [inline client/src/pages/Connect/Connect.css:7]:
```suggestion
    display: flex;
    padding-top: 50px;
    padding-bottom: 200px;
```

Comment (nathansit) [reply client/src/pages/Connect/Connect.css:7]: Some padding might be helpful for spacing, especially for the "Get Connected" button

Comment (YiyueMaggieMao) [reply client/src/pages/Connect/Connect.css:7]: Looks great! Accepted the suggestion.

File: client/src/pages/Connect/Connect.css (hunk 1 above)
Comment (YiyueMaggieMao) [reply client/src/pages/Connect/Connect.css:28]: Hmm yeah, it's something that confused me as well. Lemme see if I can do anything about this, probs after waking up tho

File: client/src/pages/Connect/Connect.js [hunk 3]
```diff
@@ -1,12 +1,140 @@
...
+                        {/* Checkboxes */}
+                        <br />
+                        <Form.Check type="checkbox" label="I'm interested in Fall 2020 Welcome Week Events" />
+                        <br />
+                        <Form.Check type="checkbox" label="I want to join a small group with 
+                    other incoming freshmen and upperclassmen" />
+                        <br />
+                        <Form.Check type="checkbox" label="I want to get involved in Klesis! (Service, Mentorship, Fun Stuff too!)" />
+                        <br />
+                        <Form.Check type="checkbox" label="I'm interested in learning about Christianity (course101.org)" />
```
Comment (nathansit) [inline client/src/pages/Connect/Connect.js:126]: It might be good to make the check boxes slightly larger to be easier to click

**Pull request merged**